### DISCUSSION TITLE
Pass if message is only emoji

### DISCUSSION THREAD

**Discussion by rlee1990** [flyerhq/flutter_chat_ui#362] [Ideas & Requests]
It would be nice to have a boolean passed down to the bubbleBuilder that lets us know if the message is only an emoji so we could hide the background

**Comment by demchenkoalex** [comment]
I think it is there already, automatically. Don't remember for sure, but someone did this feature, we have some regex to check for emojis. Maybe it works only when you have 1 emoji

**Reply by rlee1990** [reply]
It's not when you are using BubbleBuilder to create your own bubble. I added a pull request to add this.
